packages/pnpm-plugin/pnpmfile.cjs

```diff
@@ -0,0 +1,34 @@
+// @ts-check
+module.exports = {
+  hooks: {
+    /** @param {import("@pnpm/types").PnpmSettings} config */
+    updateConfig(config) {
+      // @ts-expect-error Property isn't in PnpmSettings for some reason
+      config.publicHoistPattern ??= [];
+      // @ts-expect-error Property isn't in PnpmSettings for some reason
+      config.publicHoistPattern.push(
+        '@types*',
+        'eslint',
+        'eslint-config-skuba',
+        'prettier',
+        'esbuild',
+        'jest',
+        'tsconfig-seek',
+        'typescript',
+      );
+
+      config.ignorePatchFailures = false;
+      // @ts-expect-error Property isn't in PnpmSettings for some reason
+      config.packageManagerStrictVersion = true;
```

Comment:
Similar question on whether stuff like this should be `??=` to allow consumer to override 🤔